From 6d9c9999eeb2fa51c2f5dea39b80f2dfcbba5866 Mon Sep 17 00:00:00 2001
From: Danny Baumann <dannybaumann@web.de>
Date: Mon, 26 Aug 2013 12:00:01 +0200
Subject: [PATCH 49/67] Gallery2: Fix up audio effects dialog

- Entirely remove string referencing Snapdragon, as there's nothing
  Snapdragon specific in the audio effects
- Improve margins

Change-Id: I5d2da3b16dcf60ece9b5db36cf2fc65c4a23c275
---
 res/layout/audio_effects_dialog.xml | 2 +-
 res/layout/audio_effects_title.xml  | 2 +-
 res/values/codeaurora_strings.xml   | 3 ---
 3 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/res/layout/audio_effects_dialog.xml b/res/layout/audio_effects_dialog.xml
index be72d69cd..ac7c85b82 100644
--- a/res/layout/audio_effects_dialog.xml
+++ b/res/layout/audio_effects_dialog.xml
@@ -54,7 +54,7 @@ IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <RelativeLayout
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_marginStart="12dp"
+                android:layout_marginStart="24dp"
                 android:layout_weight="1">
                 <com.android.gallery3d.ui.Knob
                     android:id="@+id/bBStrengthKnob"
diff --git a/res/layout/audio_effects_title.xml b/res/layout/audio_effects_title.xml
index 2e200d5aa..6822fdf2b 100644
--- a/res/layout/audio_effects_title.xml
+++ b/res/layout/audio_effects_title.xml
@@ -36,7 +36,7 @@ IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
     android:gravity="left|center_vertical">
 
     <TextView
-        android:text="@string/audio_effects_dialog_title"
+        android:text="@string/audio_effects"
         android:gravity="left|center_vertical"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
diff --git a/res/values/codeaurora_strings.xml b/res/values/codeaurora_strings.xml
index 540707e3a..4aa1ba275 100644
--- a/res/values/codeaurora_strings.xml
+++ b/res/values/codeaurora_strings.xml
@@ -104,9 +104,6 @@ IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
     <!-- Toast if user attempts to control audio effects without headphones plugged in. -->
     <string name="headset_plug">Plug in headphones for these effects.</string>
 
-    <!-- The title of the audio effects dialog. -->
-    <string name="audio_effects_dialog_title">Snapdragon Audio+</string>
-
     <!-- Toast message for share items -->
     <string name="cannot_share_items">Max selected items for share is 300</string>
 
-- 
2.17.1

